### ISSUE TITLE
Error with Scikit-learn version 1.5.0

### ISSUE THREAD

**Issue by PaulineMauryL** [IBM/differential-privacy-library#95]
Hello, thanks for the good work here! I am trying to update scikit-learn but I have the folowing error.

**Describe the bug**
DiffPrivLib errors for the PCA with scikit-learn version 1.5.0 (it works for scikit-learn version 1.4.0).
The error is: "Cannot fit pipeline on data because PCA._fit_full() takes 3 positional arguments but 5 were given"

**To Reproduce**
```
import pandas as pd
from sklearn.model_selection import train_test_split
from sklearn.pipeline import Pipeline

bounds = ([30.0, 13.0, 150.0, 2000.0], [65.0, 23.0, 250.0, 7000.0])
pipeline = Pipeline([("pca",  models.PCA( n_components=8, epsilon=2.0, bounds=bounds, data_norm=100),),])
df = pd.read_csv("https://raw.githubusercontent.com/mwaskom/seaborn-data/master/penguins.csv")
df = df.dropna()
x_train, x_test = train_test_split(df, test_size=0.1, random_state=1)
pipeline = pipeline.fit(x_train)
```
and the error happens "Cannot fit pipeline on data because PCA._fit_full() takes 3 positional arguments but 5 were given".

**System information (please complete the following information):**
 - Python version: 11 and 12
 - diffprivlib version: 0.6.4 (latest)
 - numpy: 1.26.2
 - scikit-learn version: 1.5.0

Do you plan to update it ? Thank you very much :)

**Comment by naoise-h** [comment]
We have just published [diffprivlib 0.6.5 to Pypi](https://pypi.org/project/diffprivlib/0.6.5/) which should solve this issue. Please reopen the issue if the problem persists.

**Comment by PaulineMauryL** [comment]
It is all good now, thank you!
